Resources/Portals/_default/Default Website.template.nl-NL.resx

```diff
@@ -247,8 +248,68 @@
   <data name="Tab.Home.Main.Blog.Tooltip">
     <value xml:space="preserve">Informatief en vermakelijk, uitdagend en educatief, het DNN community blog heeft altijd verse content. verse ideeën en nieuwe meningen. Duik er direct in. </value>
   </data>
+  <data name="Tab.Home.Main.Blogs.Link" lastModified="2020-05-10 13:36:12">
+    <value xml:space="preserve">https://dnncommunity.org/blogs</value>
+  </data>
+  <data name="Tab.Home.Main.Blogs.SubLink.Text" lastModified="2020-05-10 13:36:12">
+    <value xml:space="preserve">Bezoek de DNN blogs</value>
+  </data>
+  <data name="Tab.Home.Main.Blogs.Title" lastModified="2020-05-10 13:36:12">
+    <value xml:space="preserve">Blogs</value>
+  </data>
+  <data name="Tab.Home.Main.Blogs.Tooltip" lastModified="2020-05-10 13:36:12">
+    <value xml:space="preserve">Informtief en onderhoudend, de DNN community blogs bieden een schat aan informatie</value>
+  </data>
+  <data name="Tab.Home.Main.Code.Link" lastModified="2020-05-10 13:36:12">
+    <value xml:space="preserve">https://github.com/dnnsoftware/Dnn.Platform</value>
+  </data>
+  <data name="Tab.Home.Main.Code.SubLink.Text" lastModified="2020-05-10 13:36:12">
+    <value xml:space="preserve">Ga naar DNN op GiHub</value>
+  </data>
+  <data name="Tab.Home.Main.Code.Title" lastModified="2020-05-10 13:36:12">
+    <value xml:space="preserve">Code</value>
+  </data>
+  <data name="Tab.Home.Main.Code.Tooltip" lastModified="2020-05-10 13:36:12">
+    <value xml:space="preserve">Het Open Source DNNPlatform is volledig beschikbaar op GitHub.</value>
+  </data>
+  <data name="Tab.Home.Main.Community.Link" lastModified="2020-05-10 15:43:14">
+    <value xml:space="preserve">https://dnncommunity.org</value>
+  </data>
+  <data name="Tab.Home.Main.Community.SubLink.Text" lastModified="2020-05-10 15:43:14">
+    <value xml:space="preserve">Bezoek de DNNCommunity site</value>
+  </data>
+  <data name="Tab.Home.Main.Community.Title" lastModified="2020-05-10 15:43:14">
+    <value xml:space="preserve">DNN Community</value>
+  </data>
+  <data name="Tab.Home.Main.Community.Tooltip" lastModified="2020-05-10 15:43:14">
+    <value xml:space="preserve">De DNN COmmunity is actief. BEzoe ons en neem deel aan de activiteiten.</value>
```

Comment:
Are COmmunity and BEzoe typos or is that correct?

Reply:
Typo's, as mentioned before, the corrections/reviews come when the key/text are all there